requirements/docs-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Docs dependencies
 -r tests-requirements.txt
-setuptools>=62.0.0
 sphinx==4.5.0
+setuptools==60.0.0
```

Comment:
What is the reason for the downgrade?

Reply:
ping @aarnphm

Reply:
this has to do with the newer version of setuptools and it setup different kind of editables install. It is better to just lock setuptools for now.